docs/general/performance.md

```diff
@@ -34,6 +34,31 @@ new Chart(ctx, {
 });
 ```
 
+## Specify `min` and `max` to scales
+
+If you specifying `min` and `max`, scale does not have to look up the range from data.
```

Comment:
I might say "If you specify the `min` and `max`, the scale does not have to compute the range from the data."

Reply:
Done